loan-events/src/test-utils.js

```diff
@@ -0,0 +1,114 @@
+const { ethers } = require('forta-agent');
+
+const defaultTypeMap = {
+  uint16: 0,
+  'uint16[]': [0],
+  uint256: 0,
+  'uint256[]': [0],
+  address: ethers.constants.AddressZero,
+  'address[]': [ethers.constants.AddressZero],
+  bytes: '0xff',
+  'bytes[]': ['0xff'],
+  bytes32: 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
+  'bytes32[]': [0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF],
```

Comment:
Out of curiosity, should both of these types have default values that are strings?  Conversely, should both of these be integers specified by hex-encoded values?

Reply:
Pretty sure `ethers.js` handles both cases, but it does look like the default value for a bytes-like object in `ethers.js` is a string.